playground/src/apollo-client.js

```diff
@@ -0,0 +1,58 @@
+import omit from 'lodash/omit';
+import { RestLink } from 'apollo-link-rest';
+import { createApolloClient } from '@commercetools-frontend/application-shell';
+import { transformLocalizedStringToLocalizedField } from '@commercetools-frontend/l10n';
+
+// TODO: move to `l10n` package.
+const applyTransformedLocalizedStrings = (
+  objectWithLocalizedStrings,
+  fieldNames
+) => {
+  const transformedFieldDefinitions = fieldNames.reduce(
+    (nextTransformed, fieldName) => ({
+      ...nextTransformed,
+      [fieldName.to]: transformLocalizedStringToLocalizedField(
+        objectWithLocalizedStrings[fieldName.from]
+      ),
+    }),
+    {}
+  );
+  const namesToOmit = fieldNames.map((fieldName) => fieldName.from);
+  const objectWithouLocalizedFields = omit(
+    objectWithLocalizedStrings,
+    namesToOmit
+  );
+  return {
+    ...objectWithouLocalizedFields,
+    ...transformedFieldDefinitions,
+  };
+};
+
+const restLink = new RestLink({
+  uri: window.app.mcApiUrl,
```

Comment:
nit: maybe we could receive the URI as a parameter of the `configureApollo` function so this module don't need to rely on some global property, but maybe this is just a personal preference.

Reply:
Yes I agree, I'm also not super happy about this.

In fact, there is another PR #2723 where the function `getMcApiUrl` is exported and can be used for these cases. I bumped into other issues there though...let's see.

> maybe we could receive the URI as a parameter of the `configureApollo` function

Could you provide an example of what you had in mind?

Reply:
Sure!
Since this module is used from the `entry-point` (and we already need to access global state over there) I would expect to provide the value from there.

Here we could have something like this:
```
const createRestLink = (uri: string) => new RestLink({
  uri,
  ...
});

const configureApollo = ({ restLinkUri }: { restLinkUri: string }) => createApolloClient({
  restLink: createRestLink(restLinkUri)
});

export default configureApollo;
```

Reply:
Thanks for the example. However, this is up to the user to do it like this in their app, it's not something we need to change/expose from within appkit...right?

Reply:
Of course. 
I added this nitpick just as it seemed a best practice for me to have in our playground, but not needed at all.